features/support/run_helpers.rb

```diff
@@ -63,17 +63,18 @@ def result_has_shell_error? result
 end
 
 
-def run_shell_command command, input
+def run_shell_command command, inputs
   result = OpenStruct.new(command: command, location: Pathname.new(Dir.pwd).basename)
   command = "#{shell_overrides}; #{command} 2>&1"
+  kill = inputs.pop && true if inputs.last == '^C'
```

Comment:
If the last input is "^C", I set a `kill` flag that prevents the result from erroring. Basically we shouldn't have an error in this situation, since "the user" is choosing to abort the command.

This isn't a true process kill as you can tell. I *was* able to figure out how to kill the child process, but I couldn't get it to have any meaningful `result`.